packages/dont-crypto/package-lock.json

```diff
@@ -0,0 +1,3793 @@
+{
```

Comment:
I think this was checked in on accident?

Reply:
yea, good catch, i'll remove it